packages/create-plugin/src/utils/utils.packagejson.ts

```diff
@@ -10,8 +10,8 @@ export type PackageJson = {
   devDependencies: Record<string, string>;
   packageManager?: string;
 } & Record<string, any>;
-export function getPackageJson(): PackageJson {
-  return readJsonFile(path.join(process.cwd(), 'package.json'));
+export function getPackageJson(rootPath = process.cwd()): PackageJson {
```

Comment:
Unless I'm blind I cannot see any usage of this additional parameter. 🤔

Reply:
This is necessary, because now `getPackageJson` is being used in an additional context: the `create-plugin generate`. Therefore I need to indicate where the project root is, so the `getPackageJson` can run there. In all the other contexts, `getPackageJson` run in the same directory as the command, therefore I kept the `process.cwd` as default.

It wasn't possible to see that in the last PR because I did a mistake and forgot to pass a different `rootPath` to `getPackgeJson`. Now this is fixed. Please check the other comments.

Reply:
Ok nice. Thanks for the explanation @oshirohugo ! 👍